webrtc-stats.html

```diff
@@ -1856,21 +1855,13 @@
                 </p>
               </dd>
               <dt>
-                <dfn><code>writable</code></dfn> of type <span class=
-                "idlMemberType"><a>boolean</a></span>
-              </dt>
-              <dd>
-                <p>
-                  Has gotten ACK to an ICE request.
-                </p>
-              </dd>
-              <dt>
-                <dfn><code>readable</code></dfn> of type <span class=
-                "idlMemberType"><a>boolean</a></span>
+                <dfn><code>writable</code></dfn> of type <span class="idlMemberType"><a>boolean</a>
+                </span>
               </dt>
               <dd>
                 <p>
-                  Has gotten a valid incoming ICE request.
+                  Has received a valid STUN binding response and the consent has not expired as
+                  defined in [[!RFC7675]] section 5.1.
```

Comment:
Probably state it is True when a valid ... is received and False when consent has expired. Probably need to know when the consent expired?